ecosystem/sep-0010.md

```diff
@@ -118,7 +118,7 @@ Name      | Type          | Description
 Example:
 
 ```
-GET https://auth.example.com/?account=GCIBUCGPOHWMMMFPFTDWBSVHQRT4DIBJ7AD6BZJYDITBK2LCVBYW7HUQ
+GET https://auth.example.com?account=GCIBUCGPOHWMMMFPFTDWBSVHQRT4DIBJ7AD6BZJYDITBK2LCVBYW7HUQ
```

Comment:
Servers should handle either with or without the slash, and most web servers do not usually distinguish. Clients are likely to include or not include the slash, depending on the HTTP client.

Take the Go http client for example. The slash is always included for root paths, and then may or may not be included for sub-root paths:
https://go.dev/play/p/srqwfsW-Jdw

Reply:
I understand this, however, an alternative example to avoid confusion could be

GET <WEB_AUTH_ENDPOINT>?account=GCIBUCGPOHWMMMFPFTDWBSVHQRT4DIBJ7AD6BZJYDITBK2LCVBYW7HUQ

now i'm checking why polaris doesn't support ending slash in server/auth

Reply:
>i'm checking why polaris doesn't support ending slash in server/auth

cc @JakeUrban

Reply:
Thanks for mentioning the polaris team.